eZ/Publish/API/Repository/URLAliasService.php

```diff
@@ -130,4 +131,19 @@ public function reverseLookup(Location $location, $languageCode = null);
      * @return \eZ\Publish\API\Repository\Values\Content\URLAlias
      */
     public function load($id);
+
+    /**
+     * Refresh all system URL aliases for the given Location (and historize outdated if needed).
+     *
+     * @param \eZ\Publish\API\Repository\Values\Content\Location $location
+     * @param \eZ\Publish\API\Repository\Values\Content\Content $content pre-loaded Content for the $location
+     */
+    public function refreshSystemUrlAliasesForLocation(Location $location, Content $content = null);
```

Comment:
given we introduce `$location->getContent()` in 2.2, maybe we should:
A. Not document this (so internal argument..) and get rid of it in 7.2 merge
B. Directly document this argument as deprecated and remove in 7.2/master merge

Reply:
> given we introduce $location->getContent() in 2.2, maybe we should:
> A. Not document this (so internal argument..) and get rid of it in 7.2 merge
> B. Directly document this argument as deprecated and remove in 7.2/master merge

B. I added to PhpDoc info that it is deprecated (54ab4d0), because I don't like PhpStorm complaining about missing/erroneous doc.